packages/modules/order/bin.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path")
```

Comment:
Q: is it normal to have it in the module order?

Reply:
Ohh. Have to delete this file. Created for expirement